### PR TITLE
Feature: Remove handlebars dependency and update ontology properties tab to use plain HTML

### PR DESCRIPTION
**Pull request by SirineMhedhbi**
## Context

See : https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/341

## Changes

- [x] Remove handlebars dependecy.
- [x]  Migrate the properties tabs to Ruby and HTML.
- [x]  Use ConceptDetailsComponent to display tab.
- [x]  Fix bug in url to display classes, properties...

- Properties tab before: 
image

- Properties tab After: 
image

- URL before: 
image

- URL after: 
image

### CODE REVIEW

**Comment by syphax-bouazzouni** [comment]
There is a bug when you go to https://testportal.lirmm.fr/ontologies/INRAETHES/?p=properties it does not select by default the first one, so the page is blank
image

File: app/views/ontologies/_tab_properties.html.haml
Comment (syphax-bouazzouni) [inline app/views/ontologies/_tab_properties.html.haml:1]: you should put this file in the properties folder, and make the show action of the PropertiesController

File: config/routes.rb [hunk 1]
```diff
@@ -125,6 +125,7 @@
   get '/ontologies/:acronym/: f', to: 'ontologies#show', constraints: { purl_conceptid: /[^\/]+/ }
   match '/ontologies/:acronym/submissions/:id/edit_metadata' => 'submissions#edit_metadata', via: [:get, :post]
   get '/ontologies_filter', to:  'ontologies#ontologies_filter'
+  get '/tab_properties', to: 'ontologies#tab_properties'
```
Comment (syphax-bouazzouni) [inline config/routes.rb:128]: make it use properties#show action instead

File: app/controllers/ontologies_controller.rb [hunk 2]
```diff
@@ -404,6 +404,14 @@ def metrics_evolution
     render partial: 'ontologies/sections/metadata/metrics_evolution_graph', locals: { data: data }
   end
 
+  def tab_properties
```
Comment (syphax-bouazzouni) [inline app/controllers/ontologies_controller.rb:407]: put this instead in properties_controller as a `show` action

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]

**Comment by syphax-bouazzouni** [comment]
Another bug is when you click on some properties it does not work 

for example `member` in https://testportal.lirmm.fr/ontologies/INRAETHES/?p=properties

in you don't show a loader when clicking on a property

File: app/views/ontologies/_tab_properties.html.haml
Comment (SirineMhedhbi) [reply app/views/ontologies/_tab_properties.html.haml:1]: There is no properties file or properties controller in the project

Comment (syphax-bouazzouni) [reply app/views/ontologies/_tab_properties.html.haml:1]: add a properties controller then

**Comment by SirineMhedhbi** [comment]
@syphax-bouazzouni  Fixed 
image

**Review by syphax-bouazzouni — APPROVED** [review]
Approved, but it can be optimized someday. By doing: 
* A component for the property  tree view 
* And don't fetch all the properties for the ontologies#properties action.

**Comment by syphax-bouazzouni** [comment]
Good bye handlebars 👋

**Pull request merged**